hyp3_autorift/process.py

```diff
@@ -154,15 +173,27 @@ def process(reference, secondary, download=False, polarization='hh', orbits=None
 
     del velocity_band, browse_tif, velocity_tif
 
-    makeAsfBrowse(str(browse_file), browse_file.stem)
+    product_name = get_product_name(reference.name, secondary.name, (reference_state_vec, secondary_state_vec))
+    makeAsfBrowse(str(browse_file), product_name)
+
+    netcdf_files = glob.glob('*.nc')
```

Comment:
could move to `Path().glob()` here, depending on if you need `netcdf_files` as paths or strings